src/witan/send/adroddiad/vega_specs/lines.clj

```diff
@@ -106,13 +107,15 @@
                                             :size        150
                                             :strokewidth 0.5}}}
                             {:mark     "errorband"
-                             :encoding {:y     {:field oru :title y-title :type "quantitative"}
-                                        :y2    {:field orl}
-                                        :color {:field group :title group-title}}}
+                             :encoding {:y       {:field oru :title y-title :type "quantitative"}
+                                        :y2      {:field orl}
+                                        :color   {:field group :title group-title}
+                                        :tooltip nil}}
```

Comment:
does this need `:tooltip nil`?

Reply:
Not needed for successful transcode, but without them you are left with the default tooltips after removing the hover rule. So put these in to actually get a plot with no tooltips (and so "remove-tooltips" actually does what it says).